charts/gardener-extension-shoot-falco-service/templates/deployment.yaml

```diff
@@ -41,6 +41,7 @@ spec:
         networking.gardener.cloud/to-runtime-apiserver: allowed
         networking.gardener.cloud/to-dns: allowed
         networking.gardener.cloud/to-public-networks: allowed
+        networking.resources.gardener.cloud/to-virtual-garden-istio-ingress-istio-ingressgateway-inte-02e09: allowed
```

Comment:
I haven't seen extension using such annotation so far. Can any of the labels below achieve the same

```yaml
        networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443: allowed
```

or

```yaml
        networking.resources.gardener.cloud/to-garden-virtual-garden-kube-apiserver-tcp-443: allowed
```

?

cc @dimityrmirchev @rfranzke

Reply:
@marwinski , @lsroe will you fix this or?